docs/content/documentation/content/search.md

```diff
@@ -4,29 +4,46 @@ weight = 100
 +++
 
 Zola can build a search index from the sections and pages content to
-be used by a JavaScript library such as [elasticlunr](http://elasticlunr.com/).
+be used by a JavaScript library such as [elasticlunr](http://elasticlunr.com/) or [fuse](https://www.fusejs.io).
 
 To enable it, you only need to set `build_search_index = true` in your `config.toml` and Zola will
 generate an index for the `default_language` set for all pages not excluded from the search index.
 
 It is very important to set the `default_language` in your `config.toml` if you are writing a site not in
 English; the index building pipelines are very different depending on the language.
 
-After `zola build` or `zola serve`, you should see two files in your public directory:
-
-- `search_index.${default_language}.js`: so `search_index.en.js` for a default setup
-- `elasticlunr.min.js`
-
-If you set `index_format = "elasticlunr_json"` in your `config.toml`, a `search_index.${default_language}.json` is generated
-instead of the default `search_index.${default_language}.js`.
-
 As each site will be different, Zola makes no assumptions about your search function and doesn't provide
 the JavaScript/CSS code to do an actual search and display results. You can look at how this site
-implements it to get an idea: [search.js](https://github.com/getzola/zola/tree/master/docs/static/search.js).
+implements it (using elasticlunr) to get an idea: [search.js](https://github.com/getzola/zola/tree/master/docs/static/search.js).
 
-If you are using a language other than English, you will also need to include the corresponding JavaScript stemmer file.
-See <https://github.com/weixsong/lunr-languages#in-a-web-browser> for details.
 
 ## Configuring the search index
 In some cases, the default indexing strategy is not suitable. You can customize which fields to include and whether
 to truncate the content in the [search configuration](@/documentation/getting-started/configuration.md).
+
+## Index Formats
+
+### Elasticlunr
+
+Compatible with [elasticlunr](http://elasticlunr.com/). Also produces `elasticlunr.min.js`.
+
+```toml
+# config.toml
+[search]
+index_format = "elasticlunr_javascript" # or "elasticlunr_json"
+```
+
+If you are using a language other than English, you will also need to include the corresponding JavaScript stemmer file.
+See <https://github.com/weixsong/lunr-languages#in-a-web-browser> for details.
+
+### Fuse
+
+Compatible with [fuse.js](https://www.fusejs.io/), [tinysearch](https://github.com/tinysearch/tinysearch), and almost
+compatible with [stork](https://stork-search.net/)
```

Comment:
What does it mean "almost compatible"? Does it work with stork or not?

Reply:
stork's format is very similar but the keys are slightly different, it's toml and it wants a bit of a extra metadata. One could conceivably convert fuse_json to stork with a shell script and a copy of [`jq`](https://github.com/jqlang/jq)

I should probably either write all that in the docs or remove the reference entirely

Reply:
actually maybe i should have fuse_json have a `url` key instead of `path`, since fuse doesn't care and that will be more compatible with stork and tiny.

Reply:
oooh wait i see why `include_path` exists